### PR TITLE
CA-C1A-C8-available-while submitted

### PR DESCRIPTION
**Pull request by praneethhm**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPET-8
https://tools.hmcts.net/jira/browse/FPET-21

### Change description ###
enabled C1A C100 final and C8 doc in submitted state

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: charts/prl-cos/values.preview.template.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ java:
     IDAM_CLIENT_REDIRECT_URI: https://prl-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/oauth2/callback
     IDAM_API_URL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
     IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-    CORE_CASE_DATA_API_URL: http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    CORE_CASE_DATA_API_URL: http://ccd-data-store-api-prl-ccd-definitions-pr-403.service.core-compute-preview.internal
```
Comment (AlokDatta) [inline charts/prl-cos/values.preview.template.yaml:31]: revert this change before merge

**Review by AlokDatta — APPROVED** [review]

**Review by priteshkr — COMMENTED** [review]
Approved the PR. Changes looks fine to me

**Review by priteshkr — APPROVED** [review]
Approving the PR.Changes looks fine.
